packages/nextjs-mf/README.md

```diff
@@ -134,29 +134,30 @@ You can see it in action here: https://github.com/module-federation/module-feder
 
 ```js
 // next.config.js
-const NextFederationPlugin = require('@module-federation/nextjs-mf/NextFederationPlugin');
+// either from default
+const NextFederationPlugin = require('@module-federation/nextjs-mf').default;
+// or named export
+const NextFederationPlugin = require('@module-federation/nextjs-mf').NextFederationPlugin;
 
 module.exports = {
   webpack(config, options) {
-    if (!options.isServer) {
-      config.plugins.push(
-        new NextFederationPlugin({
-          name: 'next2',
-          remotes: {
-            next1: `next1@http://localhost:3001/_next/static/chunks/remoteEntry.js`,
-          },
-          filename: 'static/chunks/remoteEntry.js',
-          exposes: {
-            './title': './components/exposedTitle.js',
-            './checkout': './pages/checkout',
-            './pages-map': './pages-map.js',
-          },
-          shared: {
-            // whatever else
-          },
-        })
-      );
-    }
+    config.plugins.push(
```

Comment:
This configuration is not quite right. 
1. we no need to expose `pages-map` exclusively. The plugin internally exposed them by setting `exposePages` option to true.
2. remotes have a different entry point if SSR is enabled.

Please refer to one of the examples in `apps` folder for an accurate configuration.

https://github.com/module-federation/nextjs-mf/blob/70bda37744f55af849bd5c28684f42851bbf7d1f/apps/3000-home/next.config.js#L18-L58

Reply:
👍 didn't notice it was out of date as well, fixed